src/components/DpTooltipIcon/DpTooltipIcon.stories.mdx

```diff
@@ -1,3 +1,11 @@
+import { Meta } from '@storybook/addon-docs'
+import DpTooltipIcon from './DpTooltipIcon'
+
+<Meta
+    title="Components/TooltipIcon"
+    component={DpTooltipIcon}
+/>
+
 # Tooltip Icon
 
 Renders a fontawesome Icon with a tooltip.
```

Comment:
Could you add a note that this component is deprecated in favor of DpContextualHelp? Thanks!

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/463/commits/054b54fca46701534429a3c58ab9e653f5a5c5ab